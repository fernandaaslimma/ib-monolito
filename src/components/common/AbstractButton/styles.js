import styled, { css } from "styled-components";
import { rem, media } from "../../../styles/tools";
import {
  lightBlue,
  darkBlue,
  green,
  white,
  grey,
  lightRed,
  lighterRed,
  grey70
} from "../../../styles/settings";

export const ButtonTag = styled.button`
  display: flex;
  justify-content: center;
  font-family: 'Lato Bold', 'Lato';
  background: none;
  /* border-radius: ${rem(4)}; */
  border: 0;
  border-radius: 4px;
  cursor: pointer;
  font-size: ${rem(12)};
  letter-spacing: ${rem(0.2)};
  line-height: ${rem(45)};
  padding: ${props => (props.noPadding ? "0 rem(20)" : 0)};
  position: relative;
  text-align: center;
  text-decoration: none;
  /* box-shadow: 0 4px 7px 0 rgba(74, 144, 226, 0.3); */
  transition: 0.2s ease-in-out color, 0.2s ease-in-out background,
    0.2s ease-in-out box-shadow, 0.2s ease-in-out border, 0.1s transform;
  white-space: nowrap;
  width: ${props => (props.width ? rem(props.width) : "100%")};
  user-select: none;
  /* opacity: ${props =>
    props.opacity && props.disabled ? props.opacity : 1}; */

  ${({ width, isPending }) =>
    width &&
    isPending &&
    css`
      min-width: ${rem(width)} !important;
      margin-right: 0 !important;
      text-align: center;
      padding: 0 ${rem(10)} !important;

      ${media.lg(css`
        margin-right: ${rem(20)} !important;
      `)};
    `};

  ${media.md(css`
    width: ${props => (props.width ? rem(props.width) : "100%")};
  `)};

  ${media.lg(css`
    font-size: ${rem(13)};
    padding: 0 ${rem(28)};
    line-height: ${rem(42)};
  `)};

  ${({ noPadding }) =>
    noPadding &&
    media.lg(css`
      padding: 0;
    `)}

  :active {
    transform: translateY(${rem(1)});
  }

  ${({ isCallToAction }) =>
    isCallToAction &&
    css`
      background-color: ${green};
      border: solid ${rem(1)} ${green};
      box-shadow: 0 ${rem(2)} ${rem(7)} ${rem(2)} rgba(117, 214, 184, 0.39);
      color: ${white};
      font-family: "Lato Bold", Lato;

      :hover:not([disabled]) {
        background-color: ${lightBlue};
        border-color: ${lightBlue};
        box-shadow: 0 ${rem(2)} ${rem(7)} ${rem(2)} rgba(51, 204, 208, 0.39);
      }
    `};

  ${({ isWarning }) =>
    isWarning &&
    css`
      background-color: ${lightRed};
      border: solid ${rem(1)} ${lightRed};
      box-shadow: 0 ${rem(2)} ${rem(7)} ${rem(2)} rgba(233, 103, 103, 0.3);
      color: ${white};

      :hover:not([disabled]) {
        background-color: ${lighterRed};
        border-color: ${lighterRed};
      }
    `};

  ${({ isCallToAction, isWarning }) =>
    !isCallToAction &&
    !isWarning &&
    css`
      /* border: solid ${rem(1)} ${darkBlue};
      color: ${white}; */

      :hover:not([disabled]) {
        opacity: 0.7;
      }
    `};

  ${({ small }) =>
    small &&
    css`
      font-size: ${rem(11)} !important;
      height: ${rem(40)} !important;
      line-height: ${rem(40)} !important;
    `};

  ${({ disabled, loading, opacity }) =>
    disabled &&
    !loading &&
    opacity &&
    css`
      background-color: ${!opacity && grey};
      opacity: ${opacity};
      box-shadow: none;
      border: solid ${rem(1)} ${grey};
      color: ${white};
      cursor: not-allowed;
      pointer-events: none;
    `};

  ${({ disabled, loading }) =>
    disabled &&
    loading &&
    css`
      box-shadow: none;
      cursor: not-allowed;
      pointer-events: none;
    `};
  ${({ disabled, loading }) =>
    disabled &&
    !loading &&
    css`
      background-color: ${grey};
      box-shadow: none;
      border: solid ${rem(1)} ${grey};
      color: ${white};
      cursor: not-allowed;
      pointer-events: none;
    `};

  ${({ disabled, loading }) =>
    disabled &&
    loading &&
    css`
      box-shadow: none;
      cursor: not-allowed;
      pointer-events: none;
    `};

  ${({ isSecondary }) =>
    isSecondary &&
    css`
      background-color: ${white};
      border-color: ${darkBlue};
      color: ${darkBlue};
      :hover:not([disabled]) {
        background-color: ${darkBlue};
        border-color: ${darkBlue};
        color: ${white};
      }
    `}

  ${({ isSpanButton }) =>
    isSpanButton &&
    css`
      background-color: transparent;
      border: unset;
      color: ${grey70};
      box-shadow: unset;

      &:hover {
        background-color: unset !important;
        color: unset !important;
        cursor: text;
      }

      &:focus {
        outline: unset !important;
      }

      &:active {
        transform: unset !important;
      }
    `};

  ${({ heightPx }) =>
    heightPx &&
    css`
      line-height: ${rem(heightPx)};
      height: ${rem(heightPx)};
    `}
`;
