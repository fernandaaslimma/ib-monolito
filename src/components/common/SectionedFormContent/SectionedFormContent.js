import React, { useState, useRef } from "react";
import { string, bool, func } from "prop-types";
import {
  Wrapper,
  Content,
  ShowHideHead,
  Description,
  OpenEditIcon
} from "./styles";
import { Icon } from "react-bocombbm-components";

function SectionedFormContent({
  closedDescription,
  openedDescription,
  children,
  dataTest,
  valid,
  callback,
  close
}) {
  const [isShown, update] = useState(false);
  const innerRef = useRef();

  React.useEffect(() => {
    !!close && update(!close);
  }, [close]);

  const toggle = () => {
    callback && callback();
    update(!isShown);
  };

  const currentHeight = isShown
    ? innerRef.current && innerRef.current.clientHeight
    : 0;
  return (
    <Wrapper>
      <ShowHideHead
        onClick={toggle}
        isShown={isShown}
        valid={valid}
        data-test={dataTest}
      >
        <Description>
          {isShown ? openedDescription : closedDescription}
        </Description>
        {!isShown && !valid && <Icon type="Attention" width="24" height="24" />}
        <OpenEditIcon type="Arrow" width="14" height="14" />
      </ShowHideHead>
      <Content heightToCalc={currentHeight} isShown={isShown}>
        <div ref={innerRef}>{children}</div>
      </Content>
    </Wrapper>
  );
}

SectionedFormContent.defaultProps = {
  dataTest: "SectionedFormContent",
  valid: true
};

SectionedFormContent.propTypes = {
  closedDescription: string,
  openedDescription: string.isRequired,
  dataTest: string,
  valid: bool,
  callback: func
};

export default SectionedFormContent;
