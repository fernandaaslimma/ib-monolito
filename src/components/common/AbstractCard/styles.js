import styled, { css } from "styled-components";

import { white } from "../../../styles/settings";
import { rem } from "../../../styles/tools";

export const CardWrapper = styled.div`
  border-radius: ${rem(4)};
  background: ${white};
  padding: ${rem(16)} ${rem(30)};
  margin-top: ${({ marginTop }) => marginTop};
  margin-bottom: ${({ marginBottom }) => marginBottom};
  box-shadow: 0px 2px 22px 2px rgba(225, 237, 246, 0.5);
  max-width: 100%;
  position: relative;

  ${({ height }) =>
    height &&
    css`
      height: ${height};
    `};
`;
