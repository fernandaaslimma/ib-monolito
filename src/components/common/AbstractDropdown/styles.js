import styled, { css } from "styled-components";

import { rem } from "../../../styles/tools";
import { darkBlue, grey30, black50, blue20 } from "../../../styles/settings";

export const Container = styled.div`
  cursor: pointer;
  position: relative;
  width: 100%;
  overflow: hidden;
  border-radius: ${rem(4)};
  border: ${rem(1)} solid ${grey30};
  border-color: ${props => props.focus && darkBlue};
  transition: border-color 0.3s, background 0.3s;
  padding-right: ${rem(6)};

  select {
    -webkit-appearance: none !important;
  }

  ${props =>
    props.disabled &&
    css`
      background: ${blue20};
    `}

  ${props =>
    props.width &&
    css`
      max-width: ${props.width};
    `}

  ${props =>
    props.height &&
    css`
      height: ${props.height};
    `}

  svg {
    position: absolute;
    right: ${rem(12)};
    top: ${rem(19)};
  }
`;

export const DropdownWrapper = styled.select`
  cursor: pointer;
  width: 100%;
  background: transparent;
  font-size: ${rem(16)};
  // line-height: 1;
  border: 0;
  border-radius: 0;
  height: ${rem(45)};
  color: ${black50};
  padding-left: ${rem(6)};
  padding-top: ${rem(16)};
  outline: none;
  font-size: ${rem(13)};
`;

export const Label = styled.label`
  font-family: 'Lato';
  font-size: ${rem(12)};
  pointer-events: none;
  color: ${black50};
  position: absolute;
  transform: translateY(${rem(18)});
  left: ${rem(10)};
  transition: color 0.2s cubic-bezier(0, 0, 0.2, 1) 0ms,
    transform 0.2s cubic-bezier(0, 0, 0.2, 1) 0ms;
  ${props =>
    props.selected &&
    css`
      font-family: "Lato Bold";
      transform: translateY(${rem(6)});
      font-size: ${rem(11)};
    `}

  ${props =>
    props.selected &&
    props.focus &&
    css`
      color: ${darkBlue};
    `}

  ${({ filterStyle }) =>
    filterStyle &&
    css`
      font-weight: 600;
      // line-height: 14.4px;
      color: #4e768f;
      letter-spacing: ${rem(0.48)};
    `}
`;
