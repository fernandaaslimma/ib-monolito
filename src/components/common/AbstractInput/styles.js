/* eslint-disable */
import styled, { css } from "styled-components";
import { rem } from "../../../styles/tools";
import {
  white,
  grey30,
  black50,
  darkBlue,
  green,
  red,
  blue20
} from "../../../styles/settings";

export const IconContainer = styled.div`
  position: absolute;
  top: 50%;
  left: 0;
  width: ${rem(55)};
  display: flex;
  justify-content: center;
  align-items: center;
  transform: translateY(-50%);
  color: ${black50};
  transition: 0.3s color;
  pointer-events: none;
`;

export const ValidityContainer = IconContainer.extend`
  right: ${rem(-10)};
  left: auto;
  ${({ valid }) =>
    valid &&
    css`
      ${ValidityContainer} {
        color: ${green};
      }
    `};

  input:not(:focus) {
    ${({ valid }) =>
      valid === false &&
      css`
        border-color: ${red};

        ~ ${ValidityContainer}, ~ ${IconContainer}, ~ ${Label} {
          color: ${red};
        }
      `};
  }
`;

export const Tooltip = styled.div`
  visibility: visible;

  ${({ hide }) =>
    hide &&
    css`
      visibility: hidden;
    `};
`;

export const Label = styled.label`
  background-color: ${white};
  width: calc(100% - ${rem(54)});
  padding: 5px 0;
  position: absolute;
  left: ${rem(52)};
  top: 58.9%;
  transform: translateY(-50%);
  pointer-events: none;
  transition: 0.3s top, 0.2s color, 0.3s font-size;
  font-family: Lato;
  font-size: ${rem(13)};
  letter-spacing: ${rem(0.1)};
  color: ${black50};
  ${({ tinyLabels, icon }) =>
    tinyLabels &&
    !icon &&
    css`
      padding: 2px 0;
      width: calc(100% - ${rem(12)});
      font-size: ${rem(10)};
      left: ${rem(10)};
      top: 58.9%;
      transform: translateY(-50%);
    `};

  ${({ filterStyle }) =>
    filterStyle &&
    css`
      font-weight: 600;
      line-height: 14.4px;
      color: #4e768f;
      letter-spacing: ${rem(0.48)};
    `}
  ${({ disabled }) =>
    disabled &&
    css`
      background-color: ${blue20};
    `};
`;

const revealContent = css`
  padding-top: ${rem(15)};

  ~ ${Label} {
    top: ${rem(10)};
    font-family: Lato Bold;
    font-size: ${rem(11)};
    letter-spacing: ${rem(0.1)};
    text-align: left;
    color: ${darkBlue};
  }

  ~ ${IconContainer} {
    color: ${darkBlue};
  }
`;

const revealContent2 = css`
  padding-top: ${rem(15)};

  ~ ${Label} {
    top: ${rem(10)};
    font-family: Lato Bold;
    font-size: ${rem(9)};
    letter-spacing: ${rem(0.1)};
    text-align: left;
    color: ${darkBlue};
  }
`;

const revealContent3 = css`
  padding-top: ${rem(15)};

  ~ ${Label} {
    top: ${rem(12)};
    color: ${black50};
  }
`;

export const InputField = styled.input`
  width: 100%;
  height: ${rem(48)};
  ${({ heightPx }) =>
    heightPx &&
    css`
      height: ${rem(heightPx)};
    `};

  ${({ centered }) =>
    centered &&
    css`
      text-align: center;
    `};

  border-radius: ${rem(4)};
  background-color: ${white};

  ${({ disabled }) =>
    disabled &&
    css`
      background-color: ${blue20};
    `};

  border: solid ${rem(1)} ${grey30};
  font-family: Lato;
  font-size: ${rem(13)};
  letter-spacing: ${rem(0.8)};
  color: ${black50};
  padding: 0 ${rem(52)};
  transition: 0.3s padding, 0.3s border;
  box-shadow: none;

  ::-ms-clear {
    display: none !important;
  }

  :focus {
    outline: none;
    border: solid ${rem(1)} ${darkBlue};
  }

  ${({ tinyLabels }) =>
    tinyLabels &&
    css`
      padding: 0 ${rem(9.2)};
    `};

  :focus {
    ${({ tinyLabels }) => (tinyLabels ? revealContent2 : revealContent)};
  }
  ${({ isEmpty }) => !isEmpty && revealContent};
  ${({ isEmpty, tinyLabels }) => !isEmpty && tinyLabels && revealContent2};
  :not(:focus) {
    ${({ tinyLabels, isEmpty }) => tinyLabels && !isEmpty && revealContent3};
  }
`;

export const InputWrapper = styled.div`
  width: 100%;
  display: inline-block;
  position: relative;
  height: ${rem(45)};
  ${({ heightPx }) =>
    heightPx &&
    css`
      height: ${rem(heightPx)};
    `};

  ${({ valid }) =>
    valid &&
    css`
      input {
        padding-right: ${rem(30)};
        text-overflow: ellipsis;
      }
      ${ValidityContainer} {
        color: ${green};
      }
    `};

  ${({ icon }) =>
    icon &&
    css`
      input {
        padding-left: ${rem(52)};
      }
    `};

  input:not(:focus) {
    ${({ valid }) =>
      valid === false &&
      css`
        border-color: ${red};

        ~ ${ValidityContainer}, ~ ${IconContainer}, ~ ${Label} {
          color: ${red};
        }
      `};
  }
  ${({ width }) =>
    width &&
    css`
      max-width: ${width};
    `};
`;
