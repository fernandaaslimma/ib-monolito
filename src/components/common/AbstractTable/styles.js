import styled, { css } from "styled-components";

import { animationFadeIn } from "../../../styles/tools/animation";
import { blue20 } from "../../../styles/settings";

export const TableWrapper = styled.table`
  width: 100%;
  border-collapse: separate;
  border-spacing: 0 10px;
  ${animationFadeIn}

  ${({ simpleList }) =>
    simpleList &&
    css`
      border-collapse: collapse;
    `};

  ${({ marginTop }) =>
    marginTop &&
    css`
      margin-top: ${marginTop};
    `};
`;

export const TrWrapper = styled.tr`
  padding-left: 24px;
  vertical-align: middle !important;
  border-top-left-radius: 5px;
  border-bottom-left-radius: 5px;
  ${({ simpleList }) =>
    simpleList &&
    css`
      box-shadow: unset !important;

      :nth-child(2n + 1) {
        background-color: ${blue20};
      }
    `};
`;

export const TdWrapper = styled.td`
  padding: 10px 0 10px 24px
  vertical-align: middle;
  font-weight: 400;
  font-family: Lato;
  font-size: 14px;
  color: #4E768F;
  line-height: 16.8px;
  font-style: normal;
  white-space: nowrap;
  padding-left: 24px;
  

  &:first-child {
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
    padding-left: 24px;
  }

  &:last-child {
    border-bottom-right-radius: 5px;
    border-top-right-radius: 5px;
    padding-right: 24px;
  }

  ${({ textOverflow }) =>
    textOverflow &&
    css`
      & {
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
        max-width: 10px;
      }
    `};

  ${({ bold }) =>
    bold &&
    css`
      & {
        font-weight: bold;
      }
    `};

  ${({ simpleList }) =>
    simpleList &&
    css`
      &:first-child {
        border-top-left-radius: 0;
        border-bottom-left-radius: 0;
      }

      &:last-child {
        border-bottom-right-radius: 0;
        border-top-right-radius: 0;
      }
    `};
`;

export const ThWrapper = styled.th`
  font-family: "Lato";
  font-style: normal;
  font-weight: 700;
  font-size: 9px;
  line-height: 16px;
  letter-spacing: 0.4px;
  text-transform: uppercase;
  color: #27445f;
`;

export const TheadWrapper = styled.thead`
  & > tr > th {
    font-family: Lato;
    text-transform: uppercase;
    font-size: 9px;
    font-weight: bold;
    font-style: normal;
    font-stretch: normal;
    line-height: 1.78;
    letter-spacing: 0.4px;
    cursor: default;
    text-align: left;
    padding-left: 24px;
  }
`;

export const TbodyWrapper = styled.tbody`
  & > tr {
    background: white;
    box-shadow: 0 1px 12px 6px rgba(211, 225, 232, 0.13);
    transition: box-shadow 0.3s linear;
    height: 45px;

    &:hover {
      box-shadow: 0 1px 12px 6px rgba(211, 225, 232, 0.4);
    }
  }
`;
