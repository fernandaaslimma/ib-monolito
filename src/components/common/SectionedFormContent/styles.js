import styled, { css, keyframes } from "styled-components";
import {
  darkGreen,
  black60,
  white20,
  rgbaWhite20_60,
  red
} from "../../../styles/settings";
import { rem, remFontSize, media } from "../../../styles/tools";
import { Icon } from "react-bocombbm-components";
import { isInternetExplorer } from "../../../utils/getNavigator";

const showOverflow = keyframes`
  99% {
    overflow: hidden;
  }

  100% {
    overflow: visible;
  }
`;

export const Wrapper = styled.div`
  font-size: ${remFontSize(20)};
  box-shadow: ${rem(0)} ${rem(4)} ${rem(10)} ${rgbaWhite20_60};
  padding: 0;
  display: flex;
  margin-bottom: ${rem(15)};
  flex-direction: column;
  font-family: "Lato Bold", Lato;
  color: ${darkGreen};
  border: solid 1px ${rgbaWhite20_60};
  border-radius: ${rem(4)};
  background: white;
  position: relative;
`;

export const Content = styled.div`
  padding: 0;
  overflow: hidden;
  max-height: 0;
  transition: all 0.2s linear;

  ${({ heightToCalc }) =>
    heightToCalc &&
    css`
      max-height: ${heightToCalc * 6}px;
      transition: all 0.2s linear;
    `};

  ${({ isShown }) =>
    isShown &&
    css`
      animation: ${showOverflow} 0.2s linear forwards;
      ${isInternetExplorer() &&
        css`
          overflow: visible;
        `};
    `};
`;

export const ShowHideHead = styled.div`
  padding: ${rem(10)} ${rem(15)} ${rem(10)} 0;
  display: flex;
  align-items: center;
  position: relative;
  cursor: pointer;

  ${({ isShown }) =>
    isShown &&
    css`
      border-bottom: solid 1px ${white20};
      position: sticky;
      top: 0;
      z-index: 1;
      background: white;

      ${OpenEditIcon} {
        transform: rotate(-180deg);
      }

      ${Description} {
        font-family: "Lato Bold", Lato;
        font-size: ${remFontSize(13)};
      }
    `};

  ${({ valid, isShown }) =>
    !valid &&
    !isShown &&
    css`
      ${Description},
      ${Icon},
      ${OpenEditIcon} {
        color: ${red};
      }
    `};
`;

export const Description = styled.div`
  padding: 0 0 0 ${rem(14)};
  font-size: ${remFontSize(15)};
  color: ${black60};
  text-align: left;
  font-family: "Lato", Lato;
  line-height: ${rem(15)};
  ${media.sm(css`
    line-height: ${rem(28)};
  `)};
  flex-grow: 1;
`;

export const OpenEditIcon = styled(Icon)`
  transition: all 0.2s linear;
  margin-left: ${rem(10)};
`;
