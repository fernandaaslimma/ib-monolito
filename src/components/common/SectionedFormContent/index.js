import SectionedFormContent from "./SectionedFormContent";

export default SectionedFormContent;
