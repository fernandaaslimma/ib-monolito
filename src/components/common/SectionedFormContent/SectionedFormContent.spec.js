import React from "react";
import { shallow, mount } from "enzyme";

import SectionedFormContent from "./SectionedFormContent";
import { ShowHideHead, Content } from "./styles";
import { Icon } from "react-bocombbm-components";

describe("SectionedFormContent component", () => {
  let closedDescription;
  let openedDescription;

  beforeEach(() => {
    closedDescription = "mock";
    openedDescription = "mock_2";
  });

  it("should call match snapshot", () => {
    expect(
      shallow(
        <SectionedFormContent
          closedDescription={closedDescription}
          openedDescription={openedDescription}
        />
      )
    ).toMatchSnapshot();
  });

  it("should display invalid icon on head", () => {
    const wrapper = mount(
      <SectionedFormContent
        closedDescription={closedDescription}
        openedDescription={openedDescription}
        valid={false}
      />
    );

    expect(wrapper.find(Icon).length).toEqual(2);
  });

  it("should toggle exibition", () => {
    const shallowComponent = mount(
      <SectionedFormContent
        closedDescription={closedDescription}
        openedDescription={openedDescription}
      />
    );

    shallowComponent
      .find(ShowHideHead)
      .at(0)
      .simulate("click");

    expect(shallowComponent.find(Content).prop("isShown")).toBeTruthy();
  });
});
