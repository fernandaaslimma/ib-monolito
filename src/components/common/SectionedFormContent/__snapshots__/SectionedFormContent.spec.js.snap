// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionedFormContent component should call match snapshot 1`] = `
<styled.div>
  <styled.div
    data-test="SectionedFormContent"
    isShown={false}
    onClick={[Function]}
    valid={true}
  >
    <styled.div>
      mock
    </styled.div>
    <Styled(Icon)
      height="14"
      type="Arrow"
      width="14"
    />
  </styled.div>
  <styled.div
    heightToCalc={0}
    isShown={false}
  >
    <div />
  </styled.div>
</styled.div>
`;
